Userland/Libraries/LibGfx/Font/OpenType/Font.h

```diff
@@ -6,6 +6,7 @@
 
 #pragma once
 
+#include <AK/HashMap.h>
```

Comment:
This include change looks unrelated or am I missing a transient include change here?

Reply:
This was a transitive include (which the commit does not compile  without)

Reply:
Looks like it can be removed (it did appear to be needed at one point in this refactor). Though nothing is explicitly including HashMap (so it's getting it from some random include or includer)